Tests/TestingTests/Traits/ConditionTraitTests.swift

```diff
@@ -37,15 +37,27 @@ struct ConditionTraitTests {
     .disabled(if: false)
   )
   func disabledTraitIf() throws {}
-  
+
+  @Test(
```

Comment:
These should probably be marked `.hidden`. We have infrastructure for testing fixture tests and verifying they are (or are not) run.